gateleen-hook/src/main/java/org/swisspush/gateleen/hook/HookHandler.java

```diff
@@ -569,6 +577,22 @@ public boolean handle(final RoutingContext ctx) {
             }
         }
 
+        // 1. Check if the request method is GET
+        if (request.method() == HttpMethod.GET) {
+            String uri = request.uri();
```

Comment:
Extract `var requestUri = request.uri();` only once in the `handle` method. It is already done for `PUT`and `DELETE` requests. We should optimize this.